src/main/resources/templates/templateGithubDeployment.xml

```diff
@@ -0,0 +1,42 @@
+<messageML>
+    <div class="entity" data-entity-id="githubDeployment">
+        <card class="barStyle" iconSrc="${entity['githubDeployment'].iconURL}" accent="gray">
+            <header>
+                <span class="tempo-text-color--normal">Deployment on environment</span>
+                <span class="tempo-text-color--normal"><b>${entity['githubDeployment'].deployment.environment}</b></span>
+
+                <#if entity['githubDeployment'].typeEvent == 'deployment_status'>
```

Comment:
We have to check whether is an event or another. This logic has changed a little, we don't add a 'deployment_status' field anymore.